tonic-build/Cargo.toml

```diff
@@ -25,7 +25,7 @@ syn = "2.0"
 [features]
 default = ["transport", "prost"]
 prost = ["prost-build", "dep:prost-types"]
-cleanup-markdown = ["prost", "prost-build/cleanup-markdown"]
+cleanup-markdown = ["prost-build?/cleanup-markdown"]
```

Comment:
Why is the previous setup non-additive?

Reply:
The `cleanup-markdown` feature does not assume `prost` in terms of meaning, so I used the term additive here in the sense that it will remove the behavior that implicitly enables `prost` and make it possible to be compatible with other codecs in the future. In fact, it is additive at present, so there may be a more appropriate representation.

Reply:
Should there be some added documentation to make it clearer that `cleanup-markdown` will do nothing unless `prost` is also enabled?

Reply:
Updated the document.